aries-site/src/pages/templates/forms.mdx

```diff
@@ -207,6 +208,27 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <CustomizeExample />
 </Example>
 
+### Character counter
+
+**When to use**  
+In cases where the user is given an opportunity to express themselves or would likely exceed a field's character limit. Examples include some description fields, feedback, feature requests, reporting issues, etc.  
+
+**Guidance**
+- Place the character counter at the bottom right of the field.  
+- [See guidance on character limits for fields](https://design-system.hpe.design/components/tag#anatomy). If you have doubts about how many characters to limit in the field, set a generous limit so as to not frustrate the user. When possible, check analytics to see how long it is taking your users to complete this task. Longer than usual time could indicate the character limits are too few. Increase accordingly.
+- Use a numerator/denominator format. Ex. 0/128.  
+- Count down the characters as the user is typing. Ex. 52/128 if the user has typed 52 characters. 53/128 when they type an extra character, etc.  
+- The character counter should be the same color as the field title.  
+- Make the field roughly the same size as the character limit.  
+- Red highlight the field when the user meets the character limitation.  
```

Comment:
Typically we call this "Error validation" instead of red highlight.

Reply:
Error validation could look very differently in different design systems. I figure "red highlight the field" is specific and is in line with our design system.

Reply:
Maybe specify with something like "When the user meets the character limit, use the form field error validation, which appears as a red highlight."

We define a lot of the color/states by default and try to inform people in guidance of these default treatments, so my suggestion is to try to bring the connection that the red highlight is the HPE Design System default

Reply:
Yep, we try to avoid people coming up with their own "red" and instead explaining it more semantically and tied to the properties and components involved.